okhttp-dnsoverhttps/src/main/java/okhttp3/dnsoverhttps/BootstrapDns.kt

```diff
@@ -13,33 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package okhttp3.dnsoverhttps;
+package okhttp3.dnsoverhttps
 
-import java.net.InetAddress;
-import java.net.UnknownHostException;
-import java.util.List;
-import okhttp3.Dns;
+import java.net.InetAddress
+import java.net.UnknownHostException
+import okhttp3.Dns
 
 /**
  * Internal Bootstrap DNS implementation for handling initial connection to DNS over HTTPS server.
  *
  * Returns hardcoded results for the known host.
  */
-final class BootstrapDns implements Dns {
-  private final String dnsHostname;
-  private final List<InetAddress> dnsServers;
+internal class BootstrapDns(private val dnsHostname: String, private val dnsServers: List<InetAddress>) : Dns {
 
-  BootstrapDns(String dnsHostname, List<InetAddress> dnsServers) {
-    this.dnsHostname = dnsHostname;
-    this.dnsServers = dnsServers;
-  }
-
-  @Override public List<InetAddress> lookup(String hostname) throws UnknownHostException {
-    if (!this.dnsHostname.equals(hostname)) {
-      throw new UnknownHostException(
-          "BootstrapDns called for " + hostname + " instead of " + dnsHostname);
+  @Throws(UnknownHostException::class)
+  override fun lookup(hostname: String): List<InetAddress> {
+    if (this.dnsHostname != hostname) {
+      throw UnknownHostException(
+        "BootstrapDns called for $hostname instead of $dnsHostname"
```

Comment:
Our Kotlin style is here too https://github.com/square/java-code-styles

Reply:
Applied, was this comment about indent depth?

Reply:
Yup!